.circleci/config.yml

```
@@ -36,12 +36,22 @@ jobs:
     steps:
       - trivy-scan:
           image_name: certgenerator
+  functional-test:
+    machine:
+      image: ubuntu-2004:202104-01
+      resource_class: xlarge
```

This likely only needs a docker instance, not a full VM.
```suggestion
    docker:
      image: cimg/python:3.9
```

---

Oh, I see we are doing a full kind cluster. Interesting.